package-lock.json

```diff
@@ -0,0 +1,506 @@
+{
```

Comment:
Yes, removed. We don't include them for libraries, since they are not used transitively when installing dependencies. Having them in VCS for an app makes sense though to ensure the same set of dependencies between the builds and across the development workstations.